src/types/common.rs

```diff
@@ -390,17 +391,36 @@ pub enum AttachmentItem {
 #[derive(Debug, Deserialize)]
 pub struct DateTime(#[serde(with = "time::serde::iso8601")] pub time::OffsetDateTime);
 
+impl XmlSerialize for DateTime {
+    /// Serialize this `DateTime` as an XML content node, by formatting the
+    /// innder [`time::OffsetDateTime`] to an ISO 8601 compliant string.
```

Comment:
nitpick: Rust function docs generally use present indicative rather than imperative. "a `DateTime`" is more consistent with `xml_struct` docs. Everything else is just editing.

```suggestion
    /// Serializes a `DateTime` as an XML text content node by formatting the
    /// inner [`time::OffsetDateTime`] as an ISO 8601-compliant string.
```